src/docker-osx-dev

```diff
@@ -942,9 +1014,7 @@ function configure_includes {
 #
 function watch_and_sync {
   log_info "Starting docker-osx-dev file syncing"
-  configure_boot2docker_ssh_key
-  init_boot2docker
-  install_rsync_on_boot2docker
+  init_docker_host
```

Comment:
It shouldn't cause much slow down, as the command checks if `rsync` is installed before trying to install it. The benefit is that if you create a new docker VM for whatever reason, rsync will get installed on it any time you run docker-osx-dev.